src/test/java/task/DeadlineTest.java

```diff
@@ -0,0 +1,27 @@
+package task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.task.Deadline;
```

Comment:
Perhaps change the ordering of import statements for consistency? It might make it easier to browse the list.